docs/man_pages/publishing/publish.md

```diff
@@ -0,0 +1,35 @@
+publish
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder publish`    
+Additional functionality | `$ appbuilder publish [<Command>]`
+Publishing | `$ appbuilder publish [<Connection ID>] [<Username>] [<Password>] [--force -f]`
+
+### General
+  Lists all saved connections
```

Comment:
If called without parameters it acts as `$ appbuilder publish list`, which is reflected in documentation.
After a discussion with @rosen-vladimirov it was decided that `$ appbuilder publish list` should not be included in the docs